float8_experimental/float8_linear.py

```diff
@@ -168,24 +168,28 @@ def __init__(self, *args, **kwargs):
 
         self.create_buffers()
 
-        # TODO(future): user level configuration of gemms
         self.linear_mm_config = LinearMMConfig(
-            # input
+            # output
             ScaledMMConfig(
                 emulate,
-                True if not emulate else False,
+                self.config.gemm_config_output.use_fast_accum,
                 False,
                 self.config.pad_inner_dim,
             ),
-            # weight
+            # grad_input
             ScaledMMConfig(
                 emulate,
-                True if not emulate else False,
```

Comment:
this was mistakenly set to default True in https://github.com/pytorch-labs/float8_experimental/pull/315, fixing